File: src/src_platform_uwp/fmg/uwp/img/wbmp/Mine.cs
using fmg.core.img;

namespace fmg.uwp.img.wbmp {

    /// <summary> Mine image on the playing field </summary>
    public static class Mine {

        /// <summary> Mine image controller implementation for <see cref="Logo.WBmpView"/> </summary>
        public class WBmpController : Logo.WBmpController {
            public WBmpController() { LogoModel.ToMineModel(Model); }
        }

    }

}

File: FastMines_swing/src/main/java/fmg/swing/img/Mine.java
package fmg.swing.img;

import fmg.core.img.LogoModel;

/** Mine image on the playing field */
public final class Mine {
    private Mine() {}

    /** Mine image controller implementation for {@link Logo.IconView} */
    public static class IconController extends Logo.IconController {
        public IconController() { LogoModel.toMineModel(getModel()); }
    }

    /** Mine image controller implementation for {@link Logo.ImageAwtView} */
    public static class ImageAwtController extends Logo.ImageAwtController {
        public ImageAwtController() { LogoModel.toMineModel(getModel()); }
    }

}
